aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -6,15 +6,16 @@ import { useDarkMode } from '../../utils';
 
 export const ThemeModeToggle = ({ active }) => {
   const darkMode = useDarkMode();
-  const label = darkMode.value ? 'Light Mode' : 'Dark Mode';
-  const icon = darkMode.value ? <Sun /> : <Moon />;
+  const label = darkMode.value ? 'Toggle to Light Mode' : 'Toggle to Dark Mode';
```

Comment:
Is "use" more explicit and accurate? Soft opinion.

```suggestion
  const label = darkMode.value ? 'Use Light Mode' : 'Use Dark Mode';
```

Reply:
From my research and discussions with Edgar, i'll be switching 'toggle' to 'switch'